apis/vshn/v1/dbaas_vshn_keycloak.go

```diff
@@ -93,7 +93,7 @@ type VSHNKeycloakServiceSpec struct {
 	// +kubebuilder:default="/"
 	RelativePath string `json:"relativePath,omitempty"`
 
-	// +kubebuilder:validation:Enum="23"
+	// +kubebuilder:validation:Enum="23";"24"
```

Comment:
Just a hint: According to product docs we only support the latest version, not sure how to handle that in practice, though. https://products.vshn.ch/appcat/keycloak.html#_supported_versions